wlnr/game.go

```diff
@@ -0,0 +1,277 @@
+package main
+
+import (
+	"container/list"
+	"io"
+	"log"
+	"time"
+)
+
+// ID inside the Client structure which denotes the host
+const ID_HOST = 1
+
+// The used protocol version is not known since the host has not yet connected
+const VERSION_UNKNOWN = 0
+
+type Game struct {
+	// The connection (net.Conn most likely) that let us talk to the game host
+	host *Client
+
+	// Game clients and observers. For the relay there is not difference
+	// between them
+	clients *list.List
+
+	// The id the next client will get assigned
+	nextClientId uint8
+
+	// The protocol version used for communication. Set on connect of the host
+	// and has to be the same for all clients.
+	// Being able to support different versions per client might be nice
```

Comment:
Widelands never made any guarantees that different versions of the game work together over network, I do not think the relay should try to do that. Remove this sentence?